parachain/pallets/basic-channel/src/inbound/envelope.rs

```diff
@@ -22,6 +23,8 @@ pub struct Envelope {
 	pub channel: H160,
 	/// The application on Ethereum where the message originated from.
 	pub source: H160,
+	/// The user on Ethereum that authorized the source to send the message.
+	pub user: H160,
```

Comment:
In line with suggested changes in Solidity.

```suggestion
	pub account: H160,
```